addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/CcuGateway.java

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.homematic.internal.communicator;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.xml.bind.JAXBContext;
```

Comment:
Please note that JAXB is not part of Compact Profile 2, which we plan as the future execution environment in Eclipse SmartHome. So if it isn't a too big deal to replace this code by e.g. XStream (see https://www.eclipse.org/smarthome/documentation/development/bindings/dependencies.html#for-xml-processing), it would be nice.
If not, you should at least add this dependency to the MANIFEST.MF.